### DISCUSSION TITLE
Purge Line For FLSUN Super Racer

### DISCUSSION THREAD

**Discussion by ShaenNaidoo** [blackfyre/flsun.community#8] [Q&A]
I guess I'll make the second post! 

I'm trying to implement a purge line on my SR but can't seem to get it right. I've got very little experience with gcode so all I've managed to do is basically edit the purge line sequence from the Ender 3 profile on Cura. However, when I try it, the nozzle doesn't reach the build plate. But it seems to do everything else as normal. Any help on this?

This is what I have tried, but, bear in mind that I know there is probably a lot wrong with it. I was testing and reached a dead end.

```gcode
G21
G90
M82
M107 T0
M140 S{material_bed_temperature}
M104 S{material_print_temperature} T0
M190 S{material_bed_temperature}
M109 S{material_print_temperature} T0

G1 X-124 Y-30 Z0.3 F1500.0 ; Move to start position
G1 X-124 Y30.0 Z0.3 F1500.0 E15 ; Draw the first line
G1 X-123.6 Y30.0 Z0.3 F3000.0 ; Move to side a little
G1 X-123.6 Y-30 Z0.3 F1500.0 E30 ; Draw the second line
G92 E0 ; Reset Extruder
G1 Z2.0 F1500 ; Move Z Axis up little to prevent scratching of Heat Bed
G1 X5 Y20 Z0.3 F1500.0 ; Move over to prevent blob squish


G28
G92 E0
```

Thanks in advance for the help!

**Comment by blackfyre** [comment]
There's a purge line example on the Community Hub, did you try that?
https://flsun.community/guide/#priming-the-hotend

**Reply by ShaenNaidoo** [reply]
Fantastic! I haven't. Will this be alright for a 260mm bed?

**Reply by blackfyre** [reply]
It should, worst case: nothing happens 😄

**Reply by dadoblu2000** [reply]
Hello sorry if I intervene here, can you tell me how I can change the temperatures of the extruder, I would like to set the maximum temperature from 255 degrees to 275 degrees

**Reply by blackfyre** [reply]
> Hello sorry if I intervene here, can you tell me how I can change the temperatures of the extruder, I would like to set the maximum temperature from 255 degrees to 275 degrees

That's something managed by the firmware, you'll have to re-compile Marlin with the updated values. You can find the source for the SR's brand of Marlin on the FLSUN home page 😄
